django/core/checks/registry.py

```diff
@@ -17,6 +17,7 @@ class Tags(object):
     models = 'models'
     security = 'security'
     signals = 'signals'
+    staticfiles = 'staticfiles'
```

Comment:
@timgraham  Yes, I imitated how the admin system check was written.

Perhaps I keep this file like it is now and wait for another issue that stresses on the problem, before merging my PR?